TLSharp.Core/TelegramClient.cs

```diff
@@ -110,15 +112,15 @@ public bool IsUserAuthorized()
 
             var authCheckPhoneRequest = new TLRequestCheckPhone() { phone_number = phoneNumber };
             var completed = false;
-            while(!completed)
+            while (!completed)
```

Comment:
> Second, reformatting/resetting/reverting/etc takes more time for me than you as the owner

You're completely wrong here, because you already have your branch and you just need to push new commits to it, but for me it's more annoying to have to recover your branch, create my own one making sure I keep the authorship of the commit, etc.

>  I don't have the environment and the tools/settings as you do

What environment/tools/settings do I have compared to you? I probably use the same IDE as you, or a different one, but this is not related at all with the maintainership of this project (I'm actually just a contributor, so I have limited knowledge about the project, I'm just helping sochix to review PRs).

> Forth and last, It is just a PR and you got the idea, if you don't like the code, reject it and
> implement it yourself. It doesn't really matter.

It's actually code that seems to be not crossplatform, so I actually think it's not a good idea to merge it.

You have a good option on your side, though, to not need to maintain a fork of TLSharp: use the optional delegate that can be provided since this change: https://github.com/sochix/TLSharp/commit/48077961ae730f84f1624f131dd42eb70db18cb6